app/jobs/send_periodic_invite_job.rb

```diff
@@ -24,6 +25,21 @@ def send_invitation
     Invitations::SaveAndSend.call(invitation: new_invitation, check_creneaux_availability: false)
   end
 
+  def creneaux_available_for_invitation?
+    params = @invitation.link_params.symbolize_keys.slice(
+      :motif_category_short_name,
+      :departement,
+      :organisation_ids,
+      :referent_ids
+    )
+
+    Rails.cache.fetch("RetrieveCreneauAvailability/#{params.values.join('_')}", expires_in: 12.hours) do
```

Comment:
`params.values.join('_')` n'est pas assez robuste comme clé.
Par exemple ça donne la même clé pour `{ organisation_ids: [1, 2], referent_ids: [3] }` et `{ organisation_ids: [1, 2], referent_ids: [3] }`.

On peut faire:

```suggestion
      Rails.cache.fetch("RetrieveCreneauAvailability/#{params.sort.to_h.to_query}", expires_in: 12.hours) do
```

Reply:
> Par exemple ça donne la même clé pour { organisation_ids: [1, 2], referent_ids: [3] } et { organisation_ids: [1, 2], referent_ids: [3] }.

Oui c'est normal 😂 (j'imagine que tu as du te tromper mais oui je vois ce que tu veux dire 👍